docs/tutorials/using-docker-to-cross-compile.md

```diff
@@ -100,6 +100,20 @@ control panel first][shared-drives]{: .alert-link}!
 
 [shared-drives]: https://docs.docker.com/docker-for-windows/#/shared-drives
 
+{% include /style/begin-panel.html type="info" heading="Docker Toolbox" %}
+If you are using Windows 10 Home edition, you may have received the following error:
+
+`HyperV is not available on home editions. Please use Docker Toolbox.`
+
+This is fine, you can just [download Docker Toolbox instead][docker-toolbox]{: .alert-link}.
+
+Docker Toolbox however handles shared drives differently; there is no whale icon in the system tray. Instead, Docker Toolbox automatically allows sharing of your `C:\Users\myname` folder. Also, the path handling is different, you have to use paths in linux style, so the command would be
```

Comment:
At the end of this line, can you make it clear which command you are referring to? Something like "the command mentioned below would be changed to".